client/packages/common/src/intl/locales/en/programs.json

```
@@ -11,11 +11,29 @@
   "control.search.no-results-label": "No results",
   "control.search.reset-button": "Unlink matched patient",
   "control.search.searching-label": "Searching...",
+  "description.expiry": "The closest expiry date of current stock on hand",
+  "description.rnr-consumed-adjusted": "Shows the consumption adjusted to account for stock out days. =(Consumed x 30)/(30 - Stock out duration)",
+  "description.rnr-amc": "The Average Monthly Consumption is calculated using the Adjusted consumption value, using this R&R form and the previous two, if available",
+  "description.rnr-initial-balance": "The total stock on hand for this item at the beginning of the period. This is taken from the previous R&R form, if available, or calculated using the stock on hand at the period start date.",
+  "description.rnr-final-balance": "Final balance of the stock at the end of the period. =Initial balance + Received - Consumed ± Adjustments",
+  "description.rnr-maximum-quantity": "The target quantity of stock to have on hand. =AMC x 2",
+  "description.rnr-requested-quantity": "The quantity to be ordered. The calculated value is =Maximum quantity - Final balance",
```

```suggestion
  "description.rnr-requested-quantity": "The quantity to be ordered. =Maximum quantity - Final balance",
```
Just to be consistent with other fomulae